doc/TILESET.md

```diff
@@ -383,6 +383,24 @@ Weighted variations are also possible, and can be combined with rotation:
 }
 ```
 
+Variant selection is based on the graffiti's text, so the same text will always result in the same variant shown.
+
+<details>
+<summary><h5>More graffiti variants - Spoile warning!<h5></summary>
+
+It is possible to create graffitis for specifix texts. This is meant to be an easter egg for players.
```

Comment:
```suggestion
It is possible to create graffitis for specifix texts.
```
if a tileset manage to get significant coverage on this it will become very apparent that sprites relate to the actual description of the tileset